src/main/scala/strawman/collection/View.scala

```diff
@@ -250,6 +255,19 @@ object View extends IterableFactory[View] {
 
 }
 
+object ViewTransformer {
+  type VT[-A,+B] = Iterable[A] => View[B]
+  implicit class ViewTransFormerOps[A,B](val vt: VT[A,B]) {
+    def map[C](f: B => C): VT[A,C] = v => new View.Map(vt(v),f)
+    def flatMap[C](f: B => IterableOnce[C]): VT[A,C] = v => new View.FlatMap(vt(v),f)
+    def filter(f: B => Boolean): VT[A,B] = v => new View.Filter(vt(v),f)
+    def drop(n: Int): VT[A,B] = v => new View.Drop(vt(v),n)
+    def collect[C](pf: scala.PartialFunction[B,C]): VT[A,C] = v => new View.Collect(vt(v),pf)
+    def zipwithIndex: VT[A,(B,Int)] = v => new View.ZipWithIndex[B](vt(v))
+  }
```

Comment:
I think this form is more easily grasped by a larger fraction of Scala users (as it depends on a smaller more widely used set of language features).  Elegance is fine, but approachability is usually more important.